docs/Lifecycle/index.md

```diff
@@ -0,0 +1,257 @@
+# **SDL Development Lifecycle**
```

Comment:
After agreement with stakeholder.
This was just a preparation for potential use on portal.
I do not have clear requirement to post it on portal so far.
If you think that name is ambiguous, I can provide separate index.md file with ToC.